media/stylus/includes/vars.styl

```diff
@@ -54,9 +54,9 @@ $site-font-family = 'Open Sans', Arial, sans-serif;
 $site-font-family-fallback = Arial, sans-serif;
 $heading-font-family = 'Open Sans Light', 'Helvetica', Arial, sans-serif;
 $heading-font-family-fallback = 'Helvetica', Arial, sans-serif;
-$code-inline-font-family = 'Courier New', 'Andale Mono', monospace; /* inline */
+$code-inline-font-family = 'Courier', 'Andale Mono', monospace; /* inline */
 $code-block-font-family = Consolas, Monaco, 'Andale Mono', monospace; /* prism */
-$diff-font-family = 'Courier New', 'Andale Mono', monospace;
+$diff-font-family = 'Courier', 'Andale Mono', monospace;
```

Comment:
Can we do:
```
$code-inline-font-family = $diff-font-family = 'Courier', 'Andale Mono', monospace;
```
?

If not, can we put those two vars next to each other so we remember that they're essentially the same?